doc/src/tutorials/creating-packages.md

```diff
@@ -605,9 +605,21 @@ may fit your package better.
 
      * `CPLEX.jl` wraps the `CPLEX` library, which can be identified easily in a web search.
      * `MATLAB.jl` provides an interface to call the MATLAB engine from within Julia.
+
 7. Avoid naming a package closely to an existing package
      * `Websocket` is too close to `WebSockets` and can be confusing to users. Rather use a new name such as `SimpleWebsockets`.
 
+8. Avoid using a distinctive name that is already in use in a well known, unrelated project.
+     * Don't use the names `Tkinter.jl`, `TkinterGUI.jl`, etc. for a package that is unrelated
+       to the popular `tkinter` python package. Users are likely to assume a connection that
+       does not exist.
```

Comment:
```suggestion
       to the popular `tkinter` python package, even if it provides bindings to Tcl/Tk.
       A package name of `Tkinter.jl` would only be appropriate if the package used Python's
       library to accomplish its work or was spearheaded by the same community of developers.
```

Maybe state when it would be appropriate? In my mind that more directly gets at the connections that a user might assume.